var/spack/repos/builtin/packages/python/package.py

```diff
@@ -475,21 +485,46 @@ def write_easy_install_pth(self, exts):
                         "sys.__egginsert = p+len(new)\n")
 
     def activate(self, ext_pkg, **args):
+        path_view = args.get("path_view", "")
+
         ignore = self.python_ignore(ext_pkg, args)
         args.update(ignore=ignore)
 
         super(Python, self).activate(ext_pkg, **args)
 
-        exts = spack.store.layout.extension_map(self.spec)
+        # if we are activating in a filesystem view, we need to remove
+        # the link to the main easy_install.pth file prior to writing
+        # a custom one
+        if path_view:
```

Comment:
But it only ignores `easy_install.pth` files under `bin/`. I remember I had the problem that the `easy_install.pth` file under `site-packages` was linked (which I think is correct behavior for globally activated extensions as it was before).